.circleci/workflows/openlineage-hive.yml

```diff
@@ -0,0 +1,47 @@
+workflows:
+  openlineage-hive:
+    jobs:
+      - test-integration-hive:
+          filters:
+            tags:
+              only: /^[0-9]+(\.[0-9]+){2}(-rc\.[0-9]+)?$/
+          requires:
+            - build-integration-sql-java
+            - build-client-java
+      - build-integration-hive:
+          filters:
+            tags:
+              only: /^[0-9]+(\.[0-9]+){2}(-rc\.[0-9]+)?$/
+          context: << pipeline.parameters.build-context >>
+          requires:
+           - build-integration-sql-java
+           - build-client-java
+      #FIXME - jarVerification step not defined in hive gradle, for now commented out the step
```

Comment:
Is this still the case?

Reply:
yes and I think it sould be tackled as separate PR

Reply:
Ok